### ISSUE TITLE
Blade Components are not formatted

### ISSUE THREAD

**Issue by lotsofbytes** [shufo/blade-formatter#806]
### Version

1.32.10

### Template before formatting

```blade
<x-my-layout><x-slot:my-slot>
        <div>My super cool content</div>
    </x-slot:my-slot></x-my-layout>
```

### Template after formatting

```blade
<x-my-layout><x-slot:my-slot>
    <div>My super cool content</div>
  </x-slot:my-slot></x-my-layout>
```

### Expected Behaviour

```
<x-my-layout>
  <x-slot:my-slot>
    <div>My super cool content</div>
  </x-slot:my-slot>
</x-my-layout>
```

**Comment by shufo** [comment]
It seems properly formatted as expected in my environment.

```blade
 $  cat test.blade.php
<x-my-layout><x-slot:my-slot>
        <div>My super cool content</div>
    </x-slot:my-slot></x-my-layout>

 $  blade-formatter test.blade.php
<x-my-layout>
    <x-slot:my-slot>
        <div>My super cool content</div>
    </x-slot:my-slot>
</x-my-layout>
```

Maybe the option of blade-formatter or other parts of the template are affecting the results. 
Can you give more details?

**Comment by lotsofbytes** [comment]
I tried with or without my config option file but the results are the same.
Regular html elements are formatted fine (see below). It puzzles me.  Any idea to cause this ? 
Thanks, Shufo san!

Input:
```
<x-my-layout><x-slot:my-slot>
        <table><tr><td>My super cool content</td></tr></table>
    </x-slot:my-slot></x-my-layout>

```
Output:
```
<x-my-layout><x-slot:my-slot>
    <table>
      <tr>
        <td>My super cool content</td>
      </tr>
    </table>
  </x-slot:my-slot></x-my-layout>
```

**Comment by pascalbaljet** [comment]
@shufo The formatting works when the closing tag is the same as the opening tag. But [Laravel allows](https://laravel.com/docs/10.x/blade#slots) the closing tag to be just  `</x-slot>` :

```blade
<x-alert>
    <x-slot:title>
        Server Error
    </x-slot>
 
    <strong>Whoops!</strong> Something went wrong!
</x-alert>
```

This will format into:

```blade
<x-alert>
    <x-slot:title>
        Server Error
        </x-slot>

        <strong>Whoops!</strong> Something went wrong!
</x-alert>
```

**Comment by shufo** [comment]
https://github.com/shufo/blade-formatter/issues/806#issuecomment-1580858088 fixed at v1.37.1
